src/pages/posts/Post.jsx

```diff
@@ -0,0 +1,71 @@
+import "./Post.css";
+import {useParams} from "react-router-dom";
+import {Link} from "react-router-dom";
+import dateConverter from "../../helpers/dateConverter.js";
+import {ArrowCircleLeftIcon, ArrowCircleRightIcon, ClockIcon} from "@phosphor-icons/react";
+import {useEffect, useState} from "react";
+import {deleteNumberOne, pullIdNumber} from "../../helpers/apiScripts.js";
+
+function Post() {
+    const {id} = useParams();
+    const requestId = id;
```

Comment:
Het her-opslaan van een variabele die al een naam heeft (`id`) is niet zo'n puik plan :) Als je 'm wil hernoemen, doe je dat via de destructuring rules: const `{id: differentNameIWant } = useParams();`